frontend/src/modules/otherlocales/components/OtherLocales.js

```diff
@@ -43,15 +60,30 @@ export default class Locales extends React.Component<Props> {
 
         return <section className="other-locales">
             <ul>
-                { otherlocales.translations.map((translation, index) => {
+                { orderedOtherLocales.map((translation, index) => {
+                    let lastPreferred = (index === preferredCount - 1);
+
                     return <Translation
                         translation={ translation }
                         parameters={ parameters }
                         updateEditorTranslation={ updateEditorTranslation }
+                        lastPreferred={ lastPreferred }
                         key={ index }
                     />;
                 }) }
             </ul>
         </section>;
     }
 }
+
+
+const mapStateToProps = (state: Object): Props => {
+    return {
+        orderedOtherLocales: selectors.getOrderedOtherLocales(state),
+        preferredCount: selectors.getPreferredLocalesCount(state),
+        otherlocales: state.otherlocales,
+        user: state.user,
+    };
+};
+
+export default connect(mapStateToProps)(OtherLocalesBase);
```

Comment:
Here too, I believe this shouldn't be a connected component. It might make for a lot of props to pass down from `EntityDetails`, but for the sake of consistency I think we should do that for now. Maybe we will change that later on, but if we do we'll do all three tab components the same way.